include/net/sntp.h

```diff
@@ -82,6 +82,20 @@ int sntp_query(struct sntp_ctx *ctx, u32_t timeout,
  */
 void sntp_close(struct sntp_ctx *ctx);
 
+/**
+ * @brief Convenience function to query SNTP in one-shot fashion
+ *
+ * Convenience wrapper which calls getaddrinfo(), sntp_init(),
+ * sntp_request(), and sntp_close().
```

Comment:
I think 'sntp_request()' should be 'sntp_query()'

Reply:
> I think 'sntp_request()' should be 'sntp_query()'

Good catch, fixed.